src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1521,6 +1616,75 @@ public bool MoveNext()
             return true;
         }
     }
+
+    /// <summary>
+    /// The simple enumerator class is used for the range operator '..'
+    /// in expressions like 'A'..'B' | ForEach-Object { $_ }
+    /// </summary>
+    internal class CharRangeEnumerator : IEnumerator
+    {
+        private int _increment = 1;
+
+        private bool _firstElement = true;
+
+        public CharRangeEnumerator(char lowerBound, char upperBound)
+        {
+            LowerBound = lowerBound;
+            CurrentValue = LowerBound;
+            UpperBound = upperBound;
+            if (lowerBound > upperBound)
+                _increment = -1;
+        }
+
+        public string Current
+        {
+            get { return new String((char)CurrentValue, 1); }
```

Comment:
Why convert to string? I was thinking the output should be char.

Reply:
Yes, it is _Char_ enumerator. 

Fixed.